package.json

```diff
@@ -47,8 +44,11 @@
     "chokidar-cli": "^1.2.0",
     "live-server": "^0.9.2",
     "mkdirp": "^0.5.1",
+    "mofo-bootstrap": "^1.0.1",
     "node-sass": "^3.4.2",
     "npm-run-all": "^1.7.0",
+    "react": "^15.0.0",
```

Comment:
both, I guess. dev deps install unless --production is used, so does it install 15, unless it's getting installed as itself a dependency with a sibling React?